challenge-30/submissions/atplay90/solution-template.go

```
@@ -0,0 +1,156 @@
+	fmt.Println("Implement the context manager methods!")
+
+	// Example of how the context manager should work:
+	cm := NewContextManager()
+
+	// Create a cancellable context
+	ctx, cancel := cm.CreateCancellableContext(context.Background())
+	defer cancel()
+
+	// Add some values
+	ctx = cm.AddValue(ctx, "user", "alice")
+	ctx = cm.AddValue(ctx, "requestID", "12345")
```

golangci-lint (2.12.2)

[error] ineffectual assignment to ctx

(ineffassign)